src/components/BulkEditList/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/ValuesColumn.test.js

```diff
@@ -109,13 +120,27 @@ describe('ValuesColumn Component', () => {
     await waitFor(() => expect(onChange).toHaveBeenCalled());
   });
 
-  it('should render select with item note types when action type is LOAN_TYPE', async () => {
+  it('should render select with item note types when action type is NOTE_SELECT', async () => {
     const { container } = renderComponent(() => CONTROL_TYPES.NOTE_SELECT);
     const element = container.querySelector('#noteType');
 
     expect(element).toBeInTheDocument();
 
-    fireEvent.change(element, { target: { value: 'newLoanType' } });
+    fireEvent.change(element, { target: { value: 'new note select value' } });
+
+    await waitFor(() => expect(onChange).toHaveBeenCalled());
+  });
+
+  it('should render select with item note types when action type is NOTE_SELECT + HOLDINS CAPABILITY', async () => {
+    const spy = jest.spyOn(URLSearchParams.prototype, 'get');
+    spy.mockReturnValueOnce(CAPABILITIES.HOLDING);
+
+    const { container } = renderComponent(() => CONTROL_TYPES.NOTE_SELECT);
+    const element = container.querySelector('#noteHoldingsType');
```

Comment:
here the same, try to use `getByRole` instead of getting elements by id. Use https://testing-playground.com/ if you are not sure which selector to use better.

Reply:
thanks, done